### PR TITLE
[GCI 2019] [Task] Add Screenshots

### PR DESCRIPTION
**Pull request by trungnt2910**
I have added a screenshot, as well as a README.md file for Butterfly. Instead of using the README for v0.9.1, I used the contents from v1.6.1, which is available at: https://web.archive.org/web/20041207043453/http://oxben.free.fr/butterfly/Butterfly.html
(The current version is v1.6.2, however, I cannot find the source code for that, so I have to use the web archive for v1.6.1).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+#  ![Butterfly Icon](distribute/bficon.jpg) Butterfly
+
+    File: Butterfly-v1.6.pkg
```
Comment (humdingerb) [inline README.md]: I think we can remove that whole block. Just keep: "by Thomas Thery"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    Author: Thomas THERY <dragfly_kj@rocketmail.com>
+    Release: v1.6.1 (Nov 2001)
+    Compatibility: BeOS R5 for Intel
+    Description: Picture browser
+    Notes: Freeware
+
+Butterfly is a picture browser based on the translators for displaying bitmap images. It was inspired by Sea, a DOS picture browser. Its aim is to allow easy and fast browsing of an important number of pictures.
```
Comment (humdingerb) [inline README.md]: "an important number"?  Maybe "a large number"?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+## DISCLAIMER
+
+This program is provided as is. I take no responsibility for any damage or consequences directly or indirectly, intentionally or unintentionally, which may arise from using this program - YOU USE IT AT YOUR OWN RISK!
+
+
+## Features
+
+- File system browsing.
+- Drag-and-drop support (resolve symb link).
```
Comment (humdingerb) [inline README.md]:
```suggestion
- Drag-and-drop support (resolving symbolic links).
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+- Mouse wheel support to browse images.
+- Backgrounds support.
+- Multi-lingual support: English, French, Russian, Brazilian-Portuguese, Italian, Spanish, Euskera, German.
+
+
+## Usage
+
+You can either use the mouse or the keyboard to browse.
+
+To enter in a directory, double click it or press Enter. The current path is displayed in the status bar.
```
Comment (humdingerb) [inline README.md]:
```suggestion
To enter a directory, double click it or press Enter. The current path is displayed in the status bar.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+**Set as Desktop**:     Modify the current background image
+
+**Set All Desktops**:   Set all desktop background images
+
+**Zoom-in/out**:        Guess...
+
+**Preferences**:        Open the preferences panel
+
+
+## History
```
Comment (humdingerb) [inline README.md]: Can you move the entire history into a separate "HISTORY.md" file?

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (trungnt2910) [reply README.md]: I did not edit anything, all of these lines are the author's original.

**Comment by humdingerb** [comment]
Thanks

**Comment by owenca** [comment]
@humdingerb can you help review https://github.com/HaikuArchives/ExpressionEvaluator/pull/1 too? Thanks!

**Pull request merged**